.gitignore

```diff
@@ -27,4 +27,5 @@ nuget.exe
 project.lock.json
 /.vs/
 .build/
-.testPublish/
\ No newline at end of file
+.testPublish/
+launchSettings.json
```

Comment:
please don't ignore these, check them in instead. The settings matter in some of the samples